business_objects/attribute.py

```diff
@@ -116,6 +116,24 @@ def get_text_attributes(
     return {att.name: str(att.id) for att in text_attributes}
 
 
+def get_category_attributes(
+    project_id: str,
+    state_filter: List[str] = [
+        AttributeState.UPLOADED.value,
+        AttributeState.USABLE.value,
+        AttributeState.AUTOMATICALLY_CREATED.value,
+    ],
```

Comment:
antipatter in python as the default variable is only filled once so changes to it will result in unpredictable behaviour